spec/mercure.md

```diff
@@ -39,8 +39,9 @@ The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **S
 NOT**, **RECOMMENDED**, **MAY**, and **OPTIONAL**, when they appear in this document, are to be
 interpreted as described in [@!RFC2119].
 
- *  Topic: An HTTP [@!RFC7230] or HTTPS [@!RFC2818] topic URL. The unit to which one can subscribe
-    to changes.
+ *  Topic: The unit to which one can subscribe to changes. The topic **MUST** be identified by an
+    IRI [!@RFC3987] or by a string. Using an HTTPS [@!RFC2818] or HTTP [@!RFC7230] URI [@!RFC3986]
```

Comment:
```suggestion
    IRI [!@RFC3987] or by a string. Using an HTTPS [@!RFC7230] or HTTP [@!RFC7230] URI [@!RFC3986]
```

The `https` URI scheme is actually defined in https://tools.ietf.org/html/rfc7230#section-2.7.2